# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
max_vel: &max_vel 10  # 0，1，2号电机的最大转速是80r/min，差不多是2pi/s
max_vel_2: &max_vel_2 1
max_acc: &max_acc 1
# For beginners, we downscale velocity and acceleration limits.
# You can always specify higher scaling factors (<= 1.0) in your motion requests.  # Increase the values below to 1.0 to always move at maximum speed.
default_velocity_scaling_factor: 1.0
default_acceleration_scaling_factor: 1.0  # 近距离时，给0.01比较不抖
# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  base_link_to_link1:
    has_velocity_limits: true
    max_velocity: *max_vel
    has_acceleration_limits: true
    max_acceleration: *max_acc
  link1_to_link2:
    has_velocity_limits: true
    max_velocity: *max_vel
    has_acceleration_limits: true
    max_acceleration: *max_acc
  link2_to_link3:
    has_velocity_limits: true
    max_velocity: *max_vel
    has_acceleration_limits: true
    max_acceleration: *max_acc
  link3_to_link4:
    has_velocity_limits: true
    max_velocity: *max_vel
    has_acceleration_limits: true
    max_acceleration: *max_acc
  link4_to_link5:
    has_velocity_limits: true
    max_velocity: *max_vel
    has_acceleration_limits: true
    max_acceleration: *max_acc
  link5_to_gripper_link1:
    has_velocity_limits: true
    max_velocity: *max_vel
    has_acceleration_limits: true
    max_acceleration: *max_acc